### PR TITLE
Commit to resolve requests made by @Daenyth discussed after the commit.

### PR DESCRIPTION
**Pull request by woogerboy21**
See https://github.com/Cockatrice/Cockatrice/pull/1340 for details.
This should resolve everything mentioned except the following:

* Database update success check 
  * This should be updated in a future PR as additional database components outside the client ID PR should be updated as well and by implementing the check in this single commit goes against the way the other functions for handling database functionality has been written in the past.

### CODE REVIEW

**Comment by woogerboy21** [comment]
@Daenyth how's this look?

File: servatrice/src/servatrice.cpp [hunk 1]
```diff
@@ -142,8 +142,9 @@ bool Servatrice::initServer()
 {
     serverName = settingsCache->value("server/name", "My Cockatrice server").toString();
     serverId = settingsCache->value("server/id", 0).toInt();
+    clientIdRequired = settingsCache->value("server/requireclientid",0).toBool();
     bool regServerOnly = settingsCache->value("authentication/regonly", 0).toBool();
-
+
```
Comment (ZeldaZach) [inline servatrice/src/servatrice.cpp]: You added a space, mr woogerboy :stuck_out_tongue:

File: servatrice/src/servatrice.h [hunk 2]
```diff
@@ -120,7 +120,7 @@ private slots:
 	QString shutdownReason;
 	int shutdownMinutes;
 	QTimer *shutdownTimer;
```
Comment (ZeldaZach) [inline servatrice/src/servatrice.h]: This file should be tab->space'd

**Comment by Daenyth** [comment]
Looks good minus the tab/space thing. :+1:

**Pull request merged**